### PR TITLE
Reuse compliant license

### PR DESCRIPTION
**Pull request by jbigot**
Addition of a [reuse](https://reuse.software/) compliant-licensing info

### CODE REVIEW

File: cmake/FindFFTW.cmake [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+# Copyright (C) 2015, Wenzel Jakob
+# Copyright (C) 2017, Patrick Bos
+# Copyright (C) The DDC development team, see COPYRIGHT.md file
 #
-# Original version of this file:
-#   Copyright (c) 2015, Wenzel Jakob
-#   https://github.com/wjakob/layerlab/blob/master/cmake/FindFFTW.cmake, commit 4d58bfdc28891b4f9373dfe46239dda5a0b561c6
-# Modifications:
-#   Copyright (c) 2017, Patrick Bos
+# SPDX-License-Identifier: MIT
```
Comment (tpadioleau) [inline cmake/FindFFTW.cmake:5]:
```suggestion
# SPDX-License-Identifier: BSD-3-Clause
```

And add the license file to the repo

File: COPYRIGHT.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+It is distributed under the MIT License whose full text is available in the
+LICENSES/MIT.txt file.
+
+For the purpose of copyright, the DDC development team is defined as follow:
+* Centre national de la recherche scientifique (CNRS)
+* Commissariat a l'énergie atomique et aux énergies alternatives (CEA)
+* École polytechnique fédérale de Lausanne
+* Julien Bigot <julien@julien-bigot.fr>
+* Thomas Padioleau <thomas.padioleau@gmail.com>
```
Comment (tpadioleau) [inline COPYRIGHT.md:17]:
```suggestion
* Thomas Padioleau <padioleau.thomas@gmail.com>
```

**Review by tpadioleau — APPROVED** [review]
- Remove dates from copyright headers
- Remove .reuse/dep5
- Add a script to apply "reuse"

File: LICENSES/BSD-3-Clause.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>. 
+
```
Comment (tpadioleau) [inline LICENSES/BSD-3-Clause.txt:2]:
```suggestion
Copyright (C) The DDC development team, see COPYRIGHT.md file

```

Or should it be removed like in the case of the MIT license ?

File: AUTHORS [hunk 4]
```diff
@@ -1,14 +1,26 @@
+# Copyright (C) The DDC development team, see COPYRIGHT.md file
+#
+# SPDX-License-Identifier: MIT
+
 Multiple people have contributed to DDC. To show our appreciation for their
 public spirit, we list here in alphabetical order a condensed list of their
 contributions.
```
Comment (jbigot) [inline AUTHORS:8]:
```suggestion

While individual authors are listed here, they might not be the actual
copyright holders. If you are instead looking for the list of copyright
holders, please look at the COPYRIGHT.md file.

```

File: LICENSES/BSD-3-Clause.txt (hunk 3 above)
Comment (jbigot) [reply LICENSES/BSD-3-Clause.txt:2]: I don't know about removal, that's what we get with `reuse download`, but I wouldn't put the info about the copyright holder here, it's already in each and every file where it belongs. Different files might have different copyright holders.

**Review by tpadioleau — APPROVED** [review]

File: LICENSES/BSD-3-Clause.txt (hunk 3 above)
Comment (tpadioleau) [reply LICENSES/BSD-3-Clause.txt:2]: Ok let's keep it that way then

**Pull request merged**